app/javascript/controllers/disbursement_cost_form_controller.js

```diff
@@ -0,0 +1,33 @@
+// Update the DOM element with the provided total
+function updateDomElementTotal(amount, element) {
+  let formatedAmount = parseFloat(amount).toFixed(2);
+  element.innerHTML = '£' + formatedAmount;
+}
+
+// Handle change events on the form
+export function handleFormChange(event) {
+  const valueField = document.querySelector("[data-selector=valueField]");
+  const vatField = document.querySelector("[data-selector=vatField]");
```

Comment:
minor gripe - inconsistencies with use of semi-colons. I think as a rule we can agree to not use them. 

In fact we should probably have a linter fr the js code, I'll add a ticket for that